### DISCUSSION TITLE
Bug? SwinIR 4x changed to SwinIR_4x

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#1497] [General]
Hi,

`10:26:53-791395 WARNING  Could not find upscaler: SwinIR 4x using fallback: None`

img2img suddenly complains about missing upscaler, checked the settings, it changed from "SwinIR 4x" to "SwinIR_4x". So it needs a _ in the name now. After selecting that renamed version in the settings (default upscaler) it works again.

Is this an intentional change or a bug? :)

**Comment by vladmandic** [comment]
honestly, cant recall. but sampler name comes as sampler self-registeres when loaded, so not sure how it worked before.
i wouldn't call it a bug as it works as-is.
